pkgs/racket-doc/scribblings/guide/other-editors.scrbl

```diff
@@ -137,91 +137,71 @@ popular among Racketeers as well.
 @section{Vim}
 
 Many distributions of Vim ship with support for Scheme, which will mostly work
-for Racket. As of @hyperlink["https://github.com/vim/vim/commit/1aeaf8c0e0421f34e51ef674f0c9a182debe77ae"]{version 7.3.518},
-Vim detects files with the extension @tt{.rkt} as having the
-@tt{scheme} filetype. @hyperlink["https://github.com/vim/vim/commit/9cd91a1e8816d727fbdbf0b3062288e15abc5f4d"]{Version 8.2.3368}
-added support for @tt{.rktd} and @tt{.rktl}.
+for Racket. Vim also ships with some special support for Racket.
 
-In older versions, you can enable filetype detection of Racket
-files as Scheme with the following:
+The @tt{racket} filetype comes with
+@itemlist[
+  @item{syntax highlighting}
+  @item{custom indentation for Racket forms}
+  @item{and other support including comments and @tt{raco fmt}}
+]
 
-@verbatim[#:indent 2]|{
-if has("autocmd")
-  autocmd filetypedetect BufReadPost *.rkt,*.rktl,*.rktd set filetype=scheme
-endif
-}|
+There is also support for several @seclink["top" #:doc '(lib "scribblings/raco/raco.scrbl")]{raco commands}
+in the form of builtin @tt{compiler} plugins.
```

Comment:
Yes and no: not everyone knows about them, and `:help :compiler` is a decent reference. I was trying to avoid cluttering the text with more `:help` references (you'll see I removed some).

The essence is that you typically do `:compiler` before you `:make` to select the tool you want to run.

Reply:
Where will users will learn how to run Racket-specific compiler commands like `raco setup`? Ordinarily I'd say it should be in the documentation for the plugin providing this feature, but it sounds like this may be part of Vim itself.

Reply:
It is part of Vim (and the plugin): you'd do something like `:compiler raco<C-d>`, notice `racosetup`, and pick that. I think given this conversation I'll just mention `:help compiler` and leave it there